app/build.gradle

```diff
@@ -99,6 +106,8 @@ dependencies {
     implementation platform("com.google.firebase:firebase-bom:$firebaseVersion")
     implementation 'com.google.firebase:firebase-analytics-ktx'
 
+    implementation "com.google.android.gms:play-services-auth:$playServiceAuthVersion"
```

Comment:
app모듈에서 로그인 사용하는경우가 있나요?

Reply:
Hilt 컴포넌트가 app 모듈에 생성되기 때문에 @Provider 로 제공되는 구글 SDK 인스턴스를 모르면 빌드 시 에러가 발생합니다.
공식 문서를 보니 다음과 같은 기능이 있더라구요. 추가해서 해결했습니다.
>Aggregating Task
The Hilt Gradle plugin offers an option for performing Hilt’s classpath aggregation in a dedicated Gradle task. This allows the Hilt annotation processors to be isolating so they are only invoked when necessary. This reduces incremental compilation times by reducing how often an incremental change causes a rebuild of the Dagger components. Enabling this option also enables sharing test components and classpath aggregation. Note that this option replaces `enableExperimentalClasspathAggregation` since it has the same benefits without any of its caveats.

To enable the aggregating task, apply the following configuration in your Android module’s build.gradle:
```groovy
hilt {
    enableAggregatingTask = true
}
```